tests/testthat/_snaps/filter.md

```diff
@@ -3,7 +3,8 @@
     Code
       prep(rec_1, training = iris %>% slice(1:75))
     Condition
-      Error in `dplyr::filter()`:
+      Error in `step_filter()`:
+      Caused by error in `dplyr::filter()`:
```

Comment:
This together with all the other {dplyr} steps give weird errors. This are still better than before. I will try to tackle these together in a different PR latter